src/dataExtractor.js

```diff
@@ -887,16 +887,50 @@ extractors.push({
 extractors.push({
   host: 'strava.com',
   extract:
-    function (resolve) {
-      let sourceMapData = {}
-      const re = /#([0-9.]+)\/([-0-9.]+)\/([-0-9.]+)/
-      const coordArray = window.location.hash.match(re)
-      if (coordArray && coordArray.length > 3) {
-        sourceMapData.centreCoords = { 'lat': coordArray[3], 'lng': coordArray[2] }
-        sourceMapData.resolution =
-          calculateResolutionFromStdZoom(coordArray[1], coordArray[3])
+    function (resolve, reject) {
+      function urlMethod () {
+        const re = /#([0-9.]+)\/([-0-9.]+)\/([-0-9.]+)/
+        const coordArray = window.location.hash.match(re)
+        if (coordArray && coordArray.length > 3) {
+          resolve({
+            centreCoords: { lat: coordArray[3], lng: coordArray[2] },
+            resolution: calculateResolutionFromStdZoom(coordArray[1], coordArray[3])
+          })
+          return true
+        }
       }
-      resolve(sourceMapData)
+      if (urlMethod()) { return }
+
+      function tileMethod () {
+        const container = document.querySelector('.leaflet-container')
+        if (!container) { return false }
+        const containerRect = container.getBoundingClientRect()
+
+        const tile = container.querySelector('.leaflet-tile')
+        if (!tile) { return false }
+        const tileRect = tile.getBoundingClientRect()
+
+        const re = /\/([0-9]+)\/([0-9]+)\/([0-9]+)@2x.png/
```

Comment:
I tried it, and it failed to match here.
But when I removed `@2x` it did succeed (for both rides and routes).
Do you have any idea why I don't see this in the tile URL and you, I assume, do?